### PR TITLE
add powershell samples

### PR DESCRIPTION
**Pull request by HanLiMS**
Simple samples for powershell. Do we need to add comments?

### CODE REVIEW

File: APISamples/powershell/powershellSamples.ps1 [hunk 1]
```diff
@@ -0,0 +1,32 @@
+$ReportPortalUri = 'http://myserver/reports/'
```
Comment (rvpatkar) [inline APISamples/powershell/powershellSamples.ps1]: please add a comment at top providing disclaimer statement like other samples

File: APISamples/powershell/powershellSamples.ps1 [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+$item = '/test'
+$downloadPath = 'C:\download\test.rdl'
+$catalogItemsApi = $ReportPortalUri + "/api/v2.0/CatalogItems(Path='{0}')/Content/$value"
+$url = [string]::Format($catalogItemsApi, $item)
+$response = Invoke-WebRequest -Uri $url -Method Get -UseDefaultCredentials
+[System.IO.File]::WriteAllBytes($downloadPath, $response.Content)
+
+
+Write-Host "Delete an item..."
+$item = '/test'
```
Comment (rvpatkar) [inline APISamples/powershell/powershellSamples.ps1]: no param needed here

File: APISamples/powershell/powershellSamples.ps1 [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    "Content" = [System.Convert]::ToBase64String($bytes);
+    "ContentType"="";
+    "Name" = 'chart';
+    "Path" = '/chart';
+    } | ConvertTo-Json
+Invoke-WebRequest -Uri $catalogItemsUri -Method Post -Body $payload -ContentType "application/json" -UseDefaultCredentials | Out-Null
+
+
+Write-Host "Download an item..."
+$item = '/test'
```
Comment (rvpatkar) [inline APISamples/powershell/powershellSamples.ps1]: param not needed here as well

File: APISamples/powershell/powershellSamples.ps1 [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+$ReportPortalUri = 'http://myserver/reports/'
+
+Write-Host "Upload an item..."
+$uploadItemPath = 'C:\reports\test.rdl'
+$catalogItemsUri = $ReportPortalUri + "/api/v2.0/CatalogItems"
+$bytes = [System.IO.File]::ReadAllBytes($uploadItemPath)
+$payload = @{
+    "@odata.type" = "#Model.Report";
+    "Content" = [System.Convert]::ToBase64String($bytes);
```
Comment (parthsha) [inline APISamples/powershell/powershellSamples.ps1]: (Optional) You don't need ; so you can remove it.

**Pull request merged**